docs/using-injected-bindings/using-injected-bindings.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 1
+---
+
+# Using Injected Bindings
+
+The bindings could injected as files or enviroment variables. By default
+bindings are injected as files.  To inject enviroment variable, set
+`.spec.bindAsFiles` value to `false`.
+
+For determining the folder where bindings should be injected, youo can specify
+the destination using `.spec.mountPath` or you can use `SERVICE_BINDING_ROOT`
```

Comment:
that variable is set on application resource, that is important to mention.

Reply:
Done. fd3bc68deae638df60e497e2942e58bba8dcb4a2